app/assets/js/rtMedia.backbone.js

```diff
@@ -1545,11 +1546,12 @@ jQuery( document ).ready( function( $ ) {
 				$( that ).removeAttr( 'disabled' );
 
 				//Update the like counter
-				// $( '.rtmedia-like-counter' ).html( data.count );
 				if ( data.count > 0 ) {
 					$( '.rtmedia-like-info' ).removeClass( 'hide' );
+					$( '.rtm-like-comments-info' ).removeClass( 'hide' );
 				} else {
 					$( '.rtmedia-like-info' ).addClass( 'hide' );
+					$( '.rtm-like-comments-info' ).addClass( 'hide' );
```

Comment:
@Sidsector9 : Please store `$( '.rtmedia-like-info' )` & `$( '.rtm-like-comments-info' )` into variable for caching. 

And also use `$( '.rtmedia-like-info, .rtm-like-comments-info' ).removeClass( 'hide' );` to `add` or `remove` same classes for multiple elements.